### ISSUE TITLE
Card "sport + parties = snowflake"

### ISSUE THREAD

**Issue by filibert0** [SnowdaysIT/snowdays-front-end#21]
Why putting a regular snowflake icon instead of the snowdays logo that, at the moment, is not visible anywhere on the website?

**Comment by FraTheVet** [comment]
Makes sense, in fact I was in the process of creating a custom webfont for things like that (initially only containing the snowflake logo as a character, but can be expanded to yetis if ever we feel to fill the page with yetis)

Follow commits on PR #12

**Comment by FraTheVet** [comment]
Had issues creating the webfont, asked and waiting for Graphics team support

**Comment by filibert0** [comment]
Wouldn't be fine to just use an .SVG of the logo?
